src/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { UserImage } from '@ynput/ayon-react-components'
+import * as Styled from './Avatar.styled'
+
+const Avatar = ({ user }) => {
+  return (
+    <Styled.Avatar>
+        <Styled.AvatarIcon onClick={() => console.log('test')} icon='edit' />
```

Comment:
I know this is a WIP... The Icon can be a `<Button icon="edit" />` then you get built in hover effects etc

Reply:
Replaced Icon with Button, thanks for the tip. 👍